src/controller/java/AndroidDeviceControllerWrapper.cpp

```diff
@@ -92,7 +102,8 @@ AndroidDeviceControllerWrapper * AndroidDeviceControllerWrapper::AllocateNew(
 #endif
     jobject keypairDelegate, jbyteArray rootCertificate, jbyteArray intermediateCertificate, jbyteArray nodeOperationalCertificate,
     jbyteArray ipkEpochKey, uint16_t listenPort, uint16_t controllerVendorId, uint16_t failsafeTimerSeconds,
-    bool attemptNetworkScanWiFi, bool attemptNetworkScanThread, bool skipCommissioningComplete, CHIP_ERROR * errInfoOnFailure)
+    bool attemptNetworkScanWiFi, bool attemptNetworkScanThread, bool skipCommissioningComplete, jstring countryCode,
+    jint regulatoryLocationTypeValue, CHIP_ERROR * errInfoOnFailure)
```

Comment:
Do not add new arguments here. Use setters on the object to update the commissioning parameters. We cannot keep adding more args to this method

Reply:
I updated the PR to use setters and avoid additional parameters to the allocX function